### PR TITLE
Add instruction on how to use the new factories in apps

### PR DESCRIPTION
**Pull request by kennyadsl**
To load factories of extensions.

### CODE REVIEW

**Review by aldesantis — COMMENTED** [review]
I'm not sure the explanation here is very meaningful for end users. What does it mean exactly for someone to use `FactoryBot.definition_file_paths`?

**Comment by kennyadsl** [comment]
Yes, I know this could be confusing. It means that in the `spec_helper` of your host application you may have switched to the new [FactoryBot automatic loading](https://github.com/thoughtbot/factory_bot_rails#automatic-factory-definition-loading) using `FactoryBot.definition_file_paths`. Right now we can't use that API directory to load extensions factory, unless we do something like:

```ruby
FactoryBot.definition_file_paths = [
  Spree::TestingSupport::FactoryBot.definition_file_paths,
  SolidusSubscriptions::Engine.root.glob('lib/solidus_subscriptions/testing_support/factories/**/*_factory.rb').map { |path| path.sub(/.rb\z/, '') },
  Rails.root.join('spec/factories')
].flatten
```

So, we can take advantage of this method added into `solidus_dev_support`, that also loads core factories:

```ruby
SolidusDevSupport::TestingSupport::Factories.load_for(SolidusSubscription::Engine, SolidusContent::Engine)
```

This is an intermediate step, I'd like the final API in a host application to be:

```ruby
FactoryBot.definition_file_paths = [
  Spree::TestingSupport::FactoryBot.definition_file_paths,
  SolidusSubscriptions::FactoryBot.definition_file_paths,
  Rails.root.join('spec/factories')
].flatten
```

File: lib/solidus_dev_support/templates/extension/README.md [hunk 1]
```diff
@@ -45,7 +45,14 @@ When testing your application's integration with this extension you may use its
 Simply add this require statement to your spec_helper:
 
 ```ruby
-require '<%= file_name %>/factories'
+require '<%= file_name %>/testing_support/factories'
+```
+
+Or, if you are using `FactoryBot.definition_file_paths`, you can load Solidus core
+factories along with this extensions factories with this statement:
```
Comment (aldesantis) [inline lib/solidus_dev_support/templates/extension/README.md:52]: Should be "extension's factories".

**Review by aldesantis — CHANGES_REQUESTED** [review]
@kennyadsl thanks for explanation, there's a quick typo that needs to be fixed but good to go otherwise. I'm not approving because Mergify would merge it. 😅

File: lib/solidus_dev_support/templates/extension/README.md (hunk 1 above)
Comment (kennyadsl) [reply lib/solidus_dev_support/templates/extension/README.md]: Done, thanks for the review!

**Review by aldesantis — APPROVED** [review]

**Pull request merged**